playbooks/README.md

```diff
@@ -21,12 +21,20 @@ $ ansible-galaxy install -p $HOME/galaxy-roles -r requirements.yml
 $ export ANSIBLE_ROLES_PATH=$HOME/galaxy-roles
 ```
 
-For OpenShift deployment clone [**OpenShift Ansible**][openshift-ansible-project]
+### For OpenShift deployment
+
+Clone [**OpenShift Ansible**][openshift-ansible-project]:
 
 ```bash
 $ git clone -b release-3.7 https://github.com/openshift/openshift-ansible
 ```
 
+Configure path to openshift-ansible by adding following to `kubevirt-ansible/vars/all.yml`:
```

Comment:
I don't think we want to add individual vars in here. The individual vars for a playbook are found in the playbooks README.  Maybe we need to reference their location at the top level README and mention that you can overwrite any variable by including it in all.yml.